Client/Frontend/UserContent/UserScripts/BraveSkusHelper.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'use strict';
+
+window.__firefox__.includeOnce("BraveSkusHelper", function() {
+  function sendMessage(method_id, data) {
+    return webkit.messageHandlers.braveSkusHelper.postMessage({'securitytoken': '$<security_token>' ,'method_id': method_id, data: data});
+  }
+
+  if (!window.chrome) {
+    window.chrome = {};
+  }
+
+Object.defineProperty(window.chrome, 'braveSkus', {
+enumerable: false,
+configurable: true,
```

Comment:
Any reason why this is `configurable`?
The reason I ask is because a page can do:
```javascript
const braveSkus = window.chrome.braveSkus;

Object.defineProperty(window.chrome, 'braveSkus', {
enumerable: false,
configurable: false,
writable: false, 
value: {
    refresh_order(orderId) {
        console.log(orderId);
        return braveSkus.refresh_order(orderId).then(result => {
            console.log(result)
        })
    },
}
});
```
Same for the other functions. Though, it seems this is injected only on `accounts.bravesoftware.com` anyway, so should be fine.
Just curious if it can be `non-configurable` anyway.